backend/api/search.py

```diff
@@ -84,34 +84,51 @@ def search_avanced():
 
     try:
         search_string = query['searchString']
-        authors = query.get('authors', [])
-        tags = query.get('tags', [])
+        authors = query.get('authors')
+        tags = query.get('tags')
         status = query.get('status')
         date_from = query.get('date_from')
         date_to = query.get('date_to')
         description = query.get('description')
+        title = query.get('')
     except KeyError:
```

Comment:
since we use dict.get() everywhere catching KeyErrors won't work since get just returns None as default if the key doesn't exist. I think all the assignments should work with the __iter__ call instead (query['authors'] etc.)

Also title = query['']? Empty key?